moveit_setup_assistant_plugins/author_information_widget/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(author_information_widget)
```

Comment:
are you making a separate package for each widget??

i assume not... so you don't need ``project`` or things like ``add_compile_options`` as that's all inherited from base CMakeList at root of package